src/diffusers/pipelines/flux/pipeline_flux_controlnet.py

```diff
@@ -840,9 +848,12 @@ def __call__(
                 # broadcast to batch dimension in a way that's compatible with ONNX/Core ML
                 timestep = t.expand(latents.shape[0]).to(latents.dtype)
 
-                guidance = (
-                    torch.tensor([guidance_scale], device=device) if self.controlnet.config.guidance_embeds else None
-                )
+                if isinstance(self.controlnet, FluxMultiControlNetModel):
```

Comment:
this PR https://github.com/huggingface/diffusers/pull/9472 makes it possible to use a schnell base with dev controlnet, however it breaks the case when it is a FluxMultiControlNetModel because FluxMultiControlNetModel does not `config`

Reply:
I think it is save to use this : 
in controlnet_flux.py, in the foward check this:
if isinstance(self.time_text_embed, CombinedTimestepTextProjEmbeddings):
    guidance = None